src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -429,6 +430,19 @@ define(function (require, exports, module) {
         callback();
     }
 
+    // Based on http://stackoverflow.com/questions/21797299/convert-base64-string-to-arraybuffer
```

Comment:
@gideonthomas that's quite handy that the requires are cached; I've gone ahead and made a commit to port over the code from FilerFileSystem. I tested and it works for the selfie taker and image viewer. Can you review in case I missed anything?